evaluation/evaluation_readme.md

```diff
@@ -0,0 +1,13 @@
+
+
+# Evaluation for CTA-oneAPI
+
+This folder contains the scripting to evaluate the tool.
+Specifically, 2 things are done for evaluation:
+* evaluate impact on all sample projects of oneAPI-directProgramming repo ()
+* evaluate impact on real world repo relevant for the health sector, Relion ()
```

Comment:
Dose "( )" after repo and Relion prepare to contain the repo website?

Reply:
oh yes, added it. Thx!